roaring.go

```diff
@@ -251,6 +251,53 @@ func newIntIterator(a *Bitmap) *intIterator {
 	return p
 }
 
+// ManyIntIterable allows you to iterate over the values in a Bitmap
+type ManyIntIterable interface {
```

Comment:
The /Int/ in in the name makes me a bit uneasy. /ManyUint32Iterator/? What do you think? Seems quite verbose in the other hand.

Reply:
Yeah probably not ideal, but consistent with the `IntIterable` returned from `Iterator()`